backend/src/models.py

```diff
@@ -53,13 +62,11 @@ class Profile(ModelBase):
 
     def to_dict(self):
         return {
-            "riot_id": self.riot_id,
-            "riot_region": self.riot_region,
-            "user_id": self.user_id,
+            "region": self.riot_region,
             "balance": self.balance,
             "hours_played": self.hours_played,
-            "last_match_end": int(self.last_match_end.timestamp()),
-            "icon": self.icon.to_dict()
+            "icon_id": self.icon_id,
+            "last_match_end": int(self.last_match_end.timestamp()) if self.last_match_id else None
```

Comment:
**anotha nitpick:**
stiu ca teoretic e 100% ok sa faci `if self.last_match_id` (n-ar trebui sa apara meci cu id-ul 0), dar cred ca sufletul meu s-ar simti mai ok daca ar fi `if self.last_match_id is not None` xD
poti ignora acest nitpick daca vrei